app/javascript/crayons/index.js

```diff
@@ -1 +1,2 @@
+export * from '@crayons/Button';
```

Comment:
Export all buttons so that they are accessible via.

`import { Button } from '@crayons';`